pkg/selectors/kernel.go

```diff
@@ -489,6 +524,11 @@ func ParseMatchArg(k *KernelSelectorState, arg *v1alpha1.ArgSelector, sig []v1al
 		if err != nil {
 			return fmt.Errorf("writeMatchValuesInMap error: %w", err)
 		}
+	case SelectorOpSport, SelectorOpDport:
```

Comment:
What are  the type arguments that these operations can be applied to?
Can we do some checks? The type of the argument can be find via the `sig` array I believe.
It would be nice to reject policies that do not make sense at this point.

Reply:
Yep.